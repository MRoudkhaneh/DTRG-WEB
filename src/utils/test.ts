const shallowFinder = (component, attr) => component.find(`[id='${attr}']`);

export { shallowFinder };
