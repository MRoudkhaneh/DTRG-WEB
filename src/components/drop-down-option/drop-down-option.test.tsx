import { DropDownOption } from ".";
import { idFinder, classFinder, setUp, useTree } from "test";

describe("Drop down option", () => {
  let component;
  beforeEach(
    () =>
      (component = setUp(DropDownOption, {
        active: true,
        children: <p className="drop-down-option-test">test</p>,
      }))
  );
  // it("SHOUD Match <DropDownOption/> TO SNAPSHOT", () => {
  //   let dropDownOption = useTree(DropDownOption);
  //   expect(dropDownOption).toMatchSnapshot();
  // });
  // it("SHOULD RENDER WRAPPER", () => {
  //   const wrapper = idFinder(component, "drop-down-option");
  //   expect(wrapper.length).toBe(1);
  // });
  // it("SHOULD RENDER CERTIAN CLASS WHEN IS ACTIVE", () => {
  //   const wrapper = classFinder(component, ".text-indigo-500");
  //   expect(wrapper.length).toBe(1);
  // });
  // it("SHOULD RENDER CHILDREN WITH CERTIAN CLASS", () => {
  //   const wrapper = classFinder(component, ".drop-down-option-test");
  //   expect(wrapper.length).toBe(1);
  // });
  it("test", () => {
    expect(true).toBeTruthy();
  });
});
