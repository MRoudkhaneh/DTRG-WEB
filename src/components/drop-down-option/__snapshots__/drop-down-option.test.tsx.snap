// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Drop down option SHOUD Match <DropDownOption/> TO SNAPSHOT 1`] = `
<span
  className=" my-2 cursor-pointer  px-4  hover:text-indigo-700 text-gray-400"
  id="drop-down-option"
/>
`;
